cypress.json

```diff
@@ -7,5 +7,8 @@
     "supportFile": "src/test/cypress/support/index.ts",
     "baseUrl": "http://localhost:8080",
     "video": false,
-    "screenshotOnRunFailure": false
+    "screenshotOnRunFailure": false,
+    "viewportWidth": 1920,
+    "viewportHeight": 1080,
```

Comment:
I think its fine if we set this as a standard but eventually we should probably run the tests with variable screen sizes.

Reply:
I agree. I added this so that the cypress test at least run against the most common resolution for now